src/Task/TwigCs.php

```diff
@@ -26,9 +26,11 @@ public function getConfigurableOptions(): OptionsResolver
             'severity' => 'warning',
             'ruleset' => 'FriendsOfTwig\Twigcs\Ruleset\Official',
             'triggered_by' => ['twig'],
+            'exclude' => [''],
```

Comment:
don't add an empty value : just make the array empty instead

Reply:
Ok